composer.json

```diff
@@ -34,7 +34,7 @@
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "5.5.9"
+            "php": "7.0"
```

Comment:
There is no direct need for the `7.0+` version. The reason I've started adding it to projects is that my installed PHP version differs from the version targeted by the package I'm working on. By setting a platform version composer will treat everything it does as if you're on that version. This ensures that various dependency conflicts where I might install a version of something that is only for PHP `7.1` and makes the package crash on lower version (which is why also test against all supported version).

So we don't really need it, but it will prevent possible mishaps in the future.